.gitlab-ci.yml

```diff
@@ -344,8 +344,8 @@ gcp-prd:
     # Clean up even if something failed.
     - cd gcp/live/prd
     # Destroy Locust module (used for smoke tests)
-    - rake destroy_module[locust]
+    - export RAKE_REALLY_DESTROY_IN_PRD=true ; rake destroy_module[locust] ; unset RAKE_REALLY_DESTROY_IN_PRD
```

Comment:
Why not `rake destroy_module[locust] RAKE_REALLY_DESTROY_IN_PRD=true`?

Reply:
Nevermind, conversation thread was not updated under this PR and I missed your last comment.
I double checked and what I am proposing in previous comment works fine.

Reply:
Not sure I understand your comment but your proposed solution doesn't look like it will work:
https://gitlab.com/gpii-ops/gpii-infra/-/jobs/103405899

Reply:
It would work perfectly, `echo` and `rake` behave differently in this scenario.

Here is my test rake file:
```
require "rake/clean"

task :test do
  puts 'TEST_ME set to ' + ENV['TEST_ME']
end
```

And:
```
# rake test TEST_ME=raccoons 
TEST_ME set to raccoons
```

But:
```
# echo "TEST_ME set to $TEST_ME" TEST_ME=raccoons
TEST_ME set to  TEST_ME=raccoons

# TEST_ME=raccoons ; echo "TEST_ME set to $TEST_ME"
TEST_ME set to raccoons
```

Not a big deal.